### PR TITLE
improve handle-only handler

### PR DESCRIPTION
**Pull request by yomotsu**
see https://github.com/yomotsu/comparison-slider/pull/3#issuecomment-604943509

### CODE REVIEW

File: src/comparison-slider.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import './style';
 import { clamp } from './functions/math';
 import { selectorToElement } from './functions/selectorToElement';
+import { matches } from './functions/matches';
```
Comment (yomotsu) [inline src/comparison-slider.ts:4]: add IE11 support for `Element.matches`

File: src/comparison-slider.ts [hunk 2]
```diff
@@ -18,8 +19,9 @@ export default class ComparisonSlider {
 	$before: HTMLElement;
 	$after: HTMLElement;
 	$handle: HTMLElement;
+	handleOnlyControl: boolean;
```
Comment (yomotsu) [inline src/comparison-slider.ts:22]: `handleOnlyControl` should be a property of the instance

**Comment by yomotsu** [comment]
@puresick I edited your pull request.
could you take a look a little bit?

**Comment by puresick** [comment]
Looks good to me, as far as I can evaluate it with my amateurish typescript knowledge :)

**Comment by yomotsu** [comment]
this change is thanks to your PR❤️

**Pull request merged**